src/node/install.sh

```diff
@@ -185,4 +185,6 @@ if [ "${INSTALL_TOOLS_FOR_NODE_GYP}" = "true" ]; then
     fi
 fi
 
+find "${NVM_DIR}" -type d -print0 | xargs -n 1 -0 chmod g+s
```

Comment:
Why is the `chmod` on line 123 not enough?

Reply:
if you use a package manager and you are running as root, then the files/folders will not inherit those privs, for example. The `find` command helps address that.

This scenario fails when running tests in action

<img width="1217" alt="image" src="https://user-images.githubusercontent.com/24955913/194376601-0624cf1e-4bb2-4d87-b871-bb98038e8f37.png">